packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -39,16 +39,12 @@
                 :data-test-id="testIdForSection('subStatusText', index)">
                 {{ subStatusLine }}
             </div>
-            <div
+            <p
                 :class="[$style['c-stampCard1-expiryInfo']]"
                 :data-test-id="testIdForSection('expiryInfo')">
-                {{ card.expiryLine }}
-                <template v-if="hasValidExpiryDate">
-                    <span :aria-label="expiryDateAccessible">
-                        <span aria-hidden="true">{{ expiryDateVisual }}</span>
-                    </span>
-                </template>
-            </div>
+                <span aria-hidden="true">{{ card.expiryLine }}</span>
```

Comment:
Do we assume that a discount is always provided with an expiry date?

Reply:
we do make that assumption atm, but we can change that assumption easily at the adapter level.